R/cnd-entrace.R

```diff
@@ -22,6 +22,15 @@
 #' @param class A character vector of one or several classes of
 #'   conditions to be entraced.
 #'
+#' @section Inside RMarkdown documents:
+#'
+#' Call `global_entrace()` inside an RMarkdown document to cause
+#' errors and warnings to be promoted to rlang conditions that include
+#' a backtrace.
+#'
+#' This is useful in conjunction with [`rlang_backtrace_on_error`] and
+#' [`rlang_backtrace_on_warning`].
+#'
 #' @section Under the hood:
 #' On R 4.0 and newer, `global_entrace()` installs a global handler
 #' with `globalCallingHandlers()`. On older R versions, `entrace()` is
```

Comment:
I think it would be useful to include this somewhere in a copy-and-pasteable format:

```R
rlang::global_entrace()
options(
  rlang_backtrace_on_error_report = "full",
  rlang_backtrace_on_warning_report = "full"
)
```